site/docs/get-started/basic-configuration.md

```diff
@@ -31,31 +31,31 @@ The main menus are available in
     expanded when needed. You can pin a secondary or tertiary menu to
     easily navigate through its elements.
 
-![screenshot0002](/assets/images/docs/screenshot_0002.png)
+![screenshot0002](/assets/images/docs/screenshot_0002.png){:.large_img}
 
 ###  4. Change options
-Go to Settings → Configuration to change how the appliance behaves:
+In the example below, click on the upper right hand: EVM → Configuration to change how the appliance behaves:
 
 #### 1. Select the appliance to configure ####
-- ***CFME Region: Region 0*** → ***Zones*** → ***Zone: Default zone(current)*** → ***Server: EVM\[1\](current)*** should be
-        selected by default.  
--         The *hostname* and *IP address* of the appliance will be shown here, be careful as in some
-        cases this will be the internal IP address and not the one you
-        connected to.  
-
--         You can change the *Company Name* to whatever you want, the change will be reflected in the interface.  ***Change it to ManageIQ***
-
--         ***Select your time zone***, and,
-        if you want, you can change the language default.
+-       ***CFME Region: Region 0*** → ***Zones*** → ***Zone: Default zone(current)*** →
+        ***Server: EVM\[1\](current)*** should be selected by default.  
```

Comment:
I think that's too many unnecessary asterisks. Might be good to clean it up like:

-       ***CFME Region: Region 0 → Zones → Zone: Default zone(current) →
        Server: EVM\[1\](current)*** should be selected by default.